lib/oli/delivery/settings/settings_changes.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Oli.Delivery.Settings.SettingsChanges do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  schema "settings_changes" do
+    belongs_to(:resource, Oli.Resources.Resource)
+    belongs_to(:section, Oli.Delivery.Sections.Section)
+    belongs_to(:user, Oli.Accounts.User)
```

Comment:
We should change this from `belongs_to` since it creates a foreign key relationship with the `users` table, when we will sometimes be putting author ids into this.  

Just make `user_id` an integer field

Reply:
Nice catch! You are right. I will fix it.